int32 num_rovers
int32 num_boxes
geometry_msgs/Point[] initial_rover
geometry_msgs/Point[] initial_box
geometry_msgs/Point[] final_box
